packages/fxa-components/LinkExternal/index.tsx

```diff
@@ -13,9 +13,15 @@ type LinkExternalProps = {
 export const LinkExternal = ({
   className,
   href,
-  children,
+  children
 }: LinkExternalProps) => (
-  <a target="_blank" rel="noopener noreferrer" {...{ className }} {...{ href }}>
+  <a
+    data-testid="link"
```

Comment:
We may want to change this to `link-external` to better match the component name but that's just being super nit picky.